bin/nodeshift

```diff
@@ -74,6 +74,10 @@ yargs
     type: 'boolean',
     default: false
   })
+  .options('namespace', {
+    describe: 'flag to specify the project namespace to build/deploy into.  Overwrites the value from the config loader',
```

Comment:
Does it make more sense to say something like, "Overwrites any namespace settings in your OpenShift of Kubernetes configuration files"? Not sure that's an improvement, but saying "the config loader" implies knowledge of nodeshift that the user might not have.

Reply:
that makes sense.  will change